### ISSUE TITLE
GameGuru MAX - User-Guide Doesn't Cover Alpha Clipping or the Alpha Slider in the Color Picker

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [Dark-Basic-Software-Limited/GameGuruRepo#4804]
Despite it's size, the user-guide seems to be lacking when it comes to searching for certain topics. For example, MAX has a slider under Base Color called Alpha Clipping. However, if you click on Base Color to bring up the color picker, you also get a vertical slider at the far-right for alpha. Both have different purposes. But what are they? The average person probably doesn't even know the alpha slider in the color picker is even there. And even if they did, how will they learn the differences between the Alpha Clipping slider and the alpha slider in the color picker? These things need to be covered in the manual so that the end-user can look them up and discover what they're there for.

**Comment by LeeBamberTGC** [comment]
@MonkeyFrogStudio Good point, lots of secret knowledge still in MAX (it is a beast) :)  For your reference, the alpha in the color picker is the alpha component modifier of the texture (i.e. how transparent it is). The Alpha clip is a special GPU feature that will quickly hide pixels that are above the alpha clip threshold, effectively allowing cuttle cutting of textures that are more performant than setting your alpha in the texture to zero.
